fall18/p3/test.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python
+import subprocess, json, re, sys, importlib
+from inspect import getmembers, isfunction
+#import main 
+
+PASS = 'PASS'
+EPSILON = 0.0001
+
+TESTS_FUNC = [
+    ("testGetNumRecords", [], 529), #1
+    ("testGetName", [24], 'ARLENE'), #2
+    ("testGetWindSpeed", [38], 25), #3
```

Comment:
Another off by one error. Also, the test output seems confusing. Please fix.

Function test 3 (testGetWindSpeed): testGetWindSpeed(38) expected return 25 but found 30